lib/theme_check/corrector.rb

```diff
@@ -27,5 +27,10 @@ def wrap(node, insert_before, insert_after)
       line.insert(node.range[0], insert_before)
       line.insert(node.range[1] + 1 + insert_before.length, insert_after)
     end
+
+    def remove_line(node)
+      line = @template.full_line(node.line_number)
+      line.clear
```

Comment:
Will this end up removing more then we want, say if the line contains HTML?

`<p>...</p>{% assign x = 1 %}`

Worth a test case.

Reply:
Good point. I'll change my approach to account for this